Ginger/GingerCoreNET/AnalyzerLib/AnalyzerUtils.cs

```diff
@@ -55,6 +55,7 @@ public void RunSolutionAnalyzer(Solution solution, ObservableList<AnalyzerItemBa
             {
                 List<string> tempList = RunBusinessFlowAnalyzer(BF, issuesList);
                 MergeVariablesList(usedVariablesInSolution, tempList);
```

Comment:
change line to MergeVariablesList(usedVariablesInSolution, RunBusinessFlowAnalyzer(BF, issuesList)); to avoid creating tempList and clearing it for each businessflow